commands-extender/print-preview.command.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from "@angular/core";
+import { Router } from "@angular/router";
+import { Observable } from "rxjs/Observable";
+
+import { Command, ExecutionContext } from "progress-sitefinity-adminapp-sdk/app/api/v1";
+
+
+/**
+ * Serves as a command that gets invoked when the print preview button
+ * in the actions menu is clicked.
+ */
+@Injectable()
+export class PrintPreviewCommand implements Command {
+
+    /**
+     * Initializes a new instance of the PrintPreviewCommand.
+     * @param router The router that is used to navigate.
+     */
+    constructor(protected router: Router) {
```

Comment:
the constructor is on single line at other location that I saw

Reply:
which one is better ?

Reply:
single looks fine for statement less constructors